scripts/acn/helm-chart/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+description: The agents p2p DHT network
+name: agents-dht
+version: 0.0.1
+appVersion: 0.0.1
+keywords:
+- fetch
+- agents
+home: https://fetch.ai
+sources:
+- https://github.com/fetchai
+maintainers:
+- name: steevi
+  email: stefanos.malliaros@fetch.ai
+icon: https://avatars3.githubusercontent.com/u/40889903
```

Comment:
Intentionally hardcoded?

Reply:
No. I'll delete. That i copy-pasted from our infra repos, which we would like to know who contributed to which helm charts